### ISSUE TITLE
Show error line number

### ISSUE THREAD

**Issue by yutv** [Smile-SA/gdpr-dump#83]
There is few information in case of error like `The converter name is required.`. At least a line number where error occured would be very helpful.

**Comment by guvra** [comment]
You actually are supposed to get contextual information when the file validation fails, but only during the comparison between your config file and the following file:
https://github.com/Smile-SA/gdpr-dump/blob/master/app/config/schema.json

If this validation succeeds and there is a random error later, you won't have any contextual information about the error.

You should provide an example so that we can check if it's possible to improve the schema.json file.

**Comment by yutv** [comment]
Hello @guvra 

Not sure what you are trying to say with that schema.json.

The test case is the following:
1. Magento Cloud 2.4.5-p2
2. GdprDump 4.0.0
3. .gdpr-dump.yaml like below, **image you have added tens of tables like that**
```
extends: 'magento2'
version: '2.4.5'
...
tables:
  newsletter_queue:
    converters:
      newsletter_sender_name:
        condition: 'false === strpos((string) {{newsletter_sender_email}}, "@example.com")'
```
4. execute it
```
/var/www/html $ ./bin/gdpr-dump .gdpr-dump.yaml 
The converter name is required.
```

**Result:** the error is shown and you have no clue what table or what line of you config it is related to.

**Comment by guvra** [comment]
Will be fixed by [890537d](https://github.com/Smile-SA/gdpr-dump/commit/890537dd819d46c02e4d24073c8fef96f6f8e046) in next release
